src/file-events/cpp/logging.cpp

```diff
@@ -22,7 +22,7 @@ bool Logging::enabled(LogLevel level) {
         rethrowJavaException(env);
         lastLevelCheck = current;
     }
-    return minimumLogLevel <= level;
+    return minimumLogLevel <= static_cast<int>(level);
```

Comment:
```suggestion
    return minimumLogLevel <= reinterpret_cast <int>(level);
```

Do we actually need to cast here?

Reply:
We do need to cast, because `enum class` is not an `int`. The right cast to use is `static_cast`. `reinterpret_cast` is weaker, and doesn't work here: https://stackoverflow.com/a/12228616/104894